src/destructors.md

```diff
@@ -38,7 +38,7 @@ impl<T> Drop for Box<T> {
     fn drop(&mut self) {
         unsafe {
             drop_in_place(self.ptr.as_ptr());
-            Heap.dealloc(self.ptr.as_ptr() as *mut u8, Layout::new::<T>())
+            Global.dealloc(self.ptr.as_ptr() as *mut Void, Layout::new::<T>())
```

Comment:
ugh whose idea was it to make Void a thing in native rust code

Reply:
Oh sorry, I should have waited before opening this since we’re probably gonna rename Void before landing. Not sure to what, though.